packages/web/src/components/buy-audio-modal/components/StripeBuyAudioButton.tsx

```diff
@@ -47,9 +47,9 @@ export const StripeBuyAudioButton = () => {
       dispatch(
         initializeStripeModal({
           amount,
-          onrampSucceeded,
-          onrampCanceled,
-          onrampFailed: onrampCanceled,
+          onrampSucceeded: onrampSucceeded(),
+          onrampCanceled: onrampCanceled(),
+          onrampFailed: onrampCanceled(),
```

Comment:
Interesting, this was originally written to pass along the canceled action on failure...

Reply:
yeah noticed that too. was wondering if you'd notice that its weird or sthg. there is no failure action for this side of the store